update_version.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/bash
```

Comment:
```sh
sh: ./update_version.sh: /usr/bin/bash: bad interpreter: No such file or directory
```

This occur error. Remove this line and it works fine. Is this necessary?

Reply:
Without works also for me, on win and WSL.
I have commited this change